eva-server/src/main/resources/application.properties

```diff
@@ -33,3 +33,5 @@ db.collection-names.annotations=@eva.mongo.collections.annotations@
 
 management.endpoints.web.exposure.include=info,health
 management.info.git.mode=full
+
+spring.jmx.default-domain=eva.ebi.ac.uk.@timestamp@
```

Comment:
I am wondering why not keep this consistent with the approach [here](https://github.com/EBIvariation/eva-accession/blob/b3836f9e6516ab68660b958f13a7db437660ace2/eva-accession-ws/src/main/resources/application.properties#L26) and use pipe symbol?

Reply:
In the `eva-accession` project we [changed the default delimiter](https://github.com/EBIvariation/eva-accession/blob/master/pom.xml#L144) because we were having some issues but it was not the case for the `eva-ws` project

Reply:
the pipe symbol was configured for eva-accession because you can't use `@` as replacement if a property is a URL that contains `@`, which was needed for connecting to mongo through URL. 

https://github.com/EBIvariation/eva-accession/blob/master/pom.xml#L145 

https://github.com/EBIvariation/eva-accession/blob/master/eva-accession-ws/src/main/resources/application.properties#L13

Here in eva-ws we don't use the connection to mongo through URL so the default is used.